<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网络数据包抓包工具</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="catchControl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>941</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>捕获控制</string>
   </property>
   <widget class="QLabel" name="procotolLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>协议：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="procotolCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAC帧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARP包</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IP包</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP包</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TCP包</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>过滤条件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止捕获</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="startButtonWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ALL">
     <widget class="QPushButton" name="ALLStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ether">
     <widget class="QPushButton" name="EtherStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ARP">
     <widget class="QPushButton" name="ARPStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="IP">
     <widget class="QPushButton" name="IPStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="UDP">
     <widget class="QPushButton" name="UDPStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TCP">
     <widget class="QPushButton" name="TCPStartButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始捕获</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="catchList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>941</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>捕获列表</string>
   </property>
   <widget class="QTableWidget" name="catchListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>941</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>信息</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
